### DISCUSSION TITLE
Export ics-file

### DISCUSSION THREAD

**Discussion by xenix-55** [Etar-Group/Etar-Calendar#1115] [Q&A]
I'm using ETAR since I own the Google-free Phone vollaphone. Now I wanted to export my ETAR-calender in a .ics file but in the settings or preferences of ETAR I couldn't find any possibility to export date, only one to import isc files.
How can I export my calender?
I use the version 1.0.29 from f-droid.
thanks Marco

**Comment by jwhaleyca** [comment]
I use the Calendar Import-Export Android App found in the F-Droid store.  It works beautifully backing up my various calendars which contain over 4000 entries. Here's a link. Good luck!
https://f-droid.org/en/packages/org.sufficientlysecure.ical/

**Reply by tallfrye** [reply]
Why does that backup app require access to my contacts?

**Comment by hueldoeu** [comment]
i have a wish. i want a scrollable block because if i paste a link into a date and i want to remove the beginning like "https://" then i have to touch the cursor and mooooooove it to the left. this takes a lot of time.

in huawei calender i could easily scroll from up to down and in 2 seconds i am at the beginning.

**Comment by tallfrye** [comment]
Documentation implies this feature is native to the app. Is there a way to obtain backup data without a dedicated third party application?
